### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by pakrym-stripe**
Codegen for openapi 2956fca.
r? @yejia-stripe
cc @stripe/api-libraries

## Changelog
* Add support for Search API
  * Add support for `Search` method on resources `Charge`, `Customer`, `Invoice`, `PaymentIntent`, `Price`, `Product`, and `Subscription`

### CODE REVIEW

**Comment by pakrym-stripe** [comment]
PTAL @dcr-stripe ?

File: charge/client_test.go [hunk 1]
```diff
@@ -32,6 +32,19 @@ func TestChargeList(t *testing.T) {
 	assert.NotNil(t, i.ChargeList())
 }
 
+func TestChargeSearch(t *testing.T) {
+	i := Search(&stripe.ChargeSearchParams{SearchParams: stripe.SearchParams{
+		Query: "currency:USD",
```
Comment (dcr-stripe) [inline charge/client_test.go:37]: Need to wrap USD in quotes.

**Review by dcr-stripe — COMMENTED** [review]
LGTM, just one nit!

PTAL @pakrym-stripe

**Review by dcr-stripe — APPROVED** [review]
LGTM!

**Pull request merged**